lib/node_modules/@stdlib/stats/base/snanmeanpn/README.md

```diff
@@ -180,6 +168,107 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dnanmeanpn.h"
+```
+
+#### stdlib_strided_dnanmeanpn( N, \*X, strideX )
+
+Computes the [arithmetic mean][arithmetic-mean] of a single-precision floating-point strided array `x`, ignoring `NaN` values and using a two-pass error correction algorithm.
+
+```c
+const double x[] = { 1.0, -2.0, 0.0/0.0, 2.0 };
```

Comment:
These should be floats. In short, you need to fix the copy-paste mistakes throughout the C documentation here.